coq/coq.el

```diff
@@ -1203,21 +1213,45 @@ be called and no command will be sent to Coq."
      ;; If user issued a printing option then t printing.
      (and (string-match-p "\\(S\\|Uns\\)et\\s-+Printing" cmd)
           (> (length coq-last-but-one-proofstack) 0)))
-    (list "Show."))
+    (list "Show."
+    (when coq-show-proof-stepwise
+        (or
+    (when (eq coq-diffs 'off) "Show Proof.")
```

Comment:
@CyrilAnac last remark: could you add at the end of the *docstring* of this function, a remark such as the following?

```
Note: the last command added if `coq-show-proof-stepwise' is set
should match the `coq-show-proof-diffs-regexp'."
```

as it looks important to document this property somewhere (e.g. in the docstring)